assets/constants.ts

```diff
@@ -5083,6 +5083,7 @@ export default {
     "Environment": "#747142",
     "Medicine": "#657030",
     "Physics": "#58613a",
+    "Science": "#6f7a48",
```

Comment:
Curious: Why we have chosen this color value, is it standard?

Reply:
@Nik-09 I asked Sean, and he said the decisions were made with input from a designer.